### ISSUE TITLE
Bug/Feature Request Setup guideline

### ISSUE THREAD

**Issue by wurstbrot** [madhuakula/kubernetes-goat#3]
Hi,

I might miss a step.
I have minikube running. I have helm  version 2(command helm not helm2) and I follow the instructions:
* minikube start
* clone repo
* bash access-kubernetes-goat.sh
```
kubectl setup looks good.
Creating port forward for all the Kubernetes Goat resources to locally. We will be using 1230 to 1236 ports locally!
error: error executing jsonpath "{.items[0].metadata.name}": Error executing template: array index out of bounds: index 0, length 0. Printing more information for debugging the template:
	template was:
		{.items[0].metadata.name}
	object given to jsonpath engine was:
		map[string]interface {}{"apiVersion":"v1", "items":[]interface {}{}, "kind":"List", "metadata":map[string]interface {}{"resourceVersion":"", "selfLink":""}}


error: error executing jsonpath "{.items[0].metadata.name}": Error executing template: array index out of bounds: index 0, length 0. Printing more information for debugging the template:
	template was:
		{.items[0].metadata.name}
	object given to jsonpath engine was:
		map[string]interface {}{"apiVersion":"v1", "items":[]interface {}{}, "kind":"List", "metadata":map[string]interface {}{"resourceVersion":"", "selfLink":""}}


error: error executing jsonpath "{.items[0].metadata.name}": Error executing template: array index out of bounds: index 0, length 0. Printing more information for debugging the template:
	template was:
		{.items[0].metadata.name}
	object given to jsonpath engine was:
		map[string]interface {}{"apiVersion":"v1", "items":[]interface {}{}, "kind":"List", "metadata":map[string]interface {}{"resourceVersion":"", "selfLink":""}}


error: error executing jsonpath "{.items[0].metadata.name}": Error executing template: array index out of bounds: index 0, length 0. Printing more information for debugging the template:
	template was:
		{.items[0].metadata.name}
	object given to jsonpath engine was:
		map[string]interface {}{"apiVersion":"v1", "items":[]interface {}{}, "kind":"List", "metadata":map[string]interface {}{"resourceVersion":"", "selfLink":""}}


error: error executing jsonpath "{.items[0].metadata.name}": Error executing template: array index out of bounds: index 0, length 0. Printing more information for debugging the template:
	template was:
		{.items[0].metadata.name}
	object given to jsonpath engine was:
		map[string]interface {}{"apiVersion":"v1", "items":[]interface {}{}, "kind":"List", "metadata":map[string]interface {}{"resourceVersion":"", "selfLink":""}}


error: error executing jsonpath "{.items[0].metadata.name}": Error executing template: array index out of bounds: index 0, length 0. Printing more information for debugging the template:
	template was:
		{.items[0].metadata.name}
	object given to jsonpath engine was:
		map[string]interface {}{"apiVersion":"v1", "items":[]interface {}{}, "kind":"List", "metadata":map[string]interface {}{"resourceVersion":"", "selfLink":""}}


error: error executing jsonpath "{.items[0].metadata.name}": Error executing template: array index out of bounds: index 0, length 0. Printing more information for debugging the template:
	template was:
		{.items[0].metadata.name}
	object given to jsonpath engine was:
		map[string]interface {}{"apiVersion":"v1", "items":[]interface {}{}, "kind":"List", "metadata":map[string]interface {}{"resourceVersion":"", "selfLink":""}}


Visit http://127.0.0.1:1234 to get started with your Kuberenetes Goat hacking!

```
It fails because there are not pods:
```
 kubectl get pods --namespace default -l "app=build-code" 
No resources found in default namespace.

```
I assume I have to do something with helm before.

**Comment by madhuakula** [comment]
Hello @wurstbrot 

Thanks for the issue, Yeah what you said make sense. Because there were no pods running in Kubernetes cluster it's not working.

Just checking, did you run the `bash setup-kubernetes-goat.sh` script to setup the scenarios?

Thanks

**Comment by wurstbrot** [comment]
yes, I did. Now I see that it raised an error.  See https://github.com/madhuakula/kubernetes-goat/pull/5

I fixed it with:
_sudo ln -s /snap/bin/helm /bin/helm2_

Maybe we need to add that to known issues?

**Comment by madhuakula** [comment]
Make sense. Sure, I will create a doc and add this to known issues.

Thanks again!

**Comment by madhuakula** [comment]
I will close this for now  and add the docs and push the changes to docs.

Merged #5 

Thank you so much once again!
